net/pfSense-pkg-frr/files/usr/local/pkg/frr.inc

```diff
@@ -617,11 +649,12 @@ function frr_generate_config_vtysh() {
 		return;
 	}
 	if (empty($frr_conf['enable'])) {
-		/* FRR master switch is off. */
+		log_error("FRR vtysh: FRR master switch is off.");
```

Comment:
Why log this as an error? It's not a fatal problem, someone could be making configuration changes with the service disabled, and this would spam the log and not be helpful.

Reply:
I couldn't see another way to add some informational logging to the syslog to make it easily visible, but as you say it's become rather chatty. I've changed it to use function wrapping `syslog()` and added a setting to allow the user to set the package log level as you suggested, only two settings for now.